Chiliwithchili.txt

```diff
@@ -0,0 +1,21 @@
+## The Best Classic Chili 
+This traditional chili recipe is just like mom used to make with ground beef, beans, and a simple homemade blend of chili seasonings.
+
+## Instructions
+Heat olive oil in a large soup pot. 
+Add chopped onion and cook for 5 minutes.
+Add ground beef and break it apart with a wooden spoon.
+Cook for 6-7 minutes.
+
+Add chili powder, cumin, sugar, tomato paste, garlic powder, salt, pepper, and optional cayenne. 
+Stir until well combined.
+
+Required toppings
+shredded chedder cheese
+
+Optional toppings
+sour cream
+sliced green onions
+saltine crackers or oyster crackers
+diced avocado
+pepper
```

Comment:
I think we could need more optional toppings. Perhaps change

```suggestion
pepper
jalapeños
salt
bread
```

What do you think?